client/src/commands/createSmartContractProjectCommand.ts

```diff
@@ -83,6 +84,30 @@ export async function createFabricProject(): Promise<void> {
         }
     }
 
+    let chaincodeLanguageOptions: string[];
+    let chaincodeLanguage: string;
+    outputAdapter.log('Getting chaincode languages...');
+    try {
+        chaincodeLanguageOptions = await getChaincodeLanguageOptions();
+    } catch (error) {
+        console.log('creating default javascript smart contract project');
+        window.showInformationMessage('Unable to determine chaincode language options, using javascript as default');
+        chaincodeLanguage = 'javascript';
+    }
+    const choseChaincodeLanguageQuickPickOptions = {
+        placeHolder: 'Chose chaincode language (Esc to cancel)',
+        ignoreFocusOut: true,
+        matchOnDetail: true
+    };
+    if (!chaincodeLanguage) {
+        chaincodeLanguage = await window.showQuickPick(chaincodeLanguageOptions, choseChaincodeLanguageQuickPickOptions);
+        if (!chaincodeLanguageOptions.includes(chaincodeLanguage)) {
+            return;
```

Comment:
It looks like there's a default of 'javascript' if the available options cannot be determined (even though javascript may no longer be a valid option) but no default if the user picks something that isn't supported (even though a default would be safe in that case). Is that right/intentional?

Reply:
I've removed the javascript by default stuff now :)